include/industry_standard/spdm.h

```diff
@@ -508,7 +508,7 @@ typedef struct {
     uint16_t length;
 } spdm_get_certificate_request_t;
 
-#define SPDM_GET_CERTIFICATE_REQUEST_SLOT_ID_MASK 0xF
+#define SPDM_GET_CERTIFICATE_REQUEST_SLOT_ID_MASK         0xF
```

Comment:
why we need this change?

Reply:
Sorry.   The reason for change is from rebase operation. I have fixed the typo.